tslint.json

```diff
@@ -35,7 +35,7 @@
     "no-arg": true,
     "no-bitwise": true,
     "no-console": [
-      true,
+      false,
```

Comment:
I think we should keep this as-is, it's fine to use `console.log` for debugging purposes, but I'd like to start trimming out logging to the console regularly, and this setting at least ensures you've gotta switch over to logging or take out log statements.

Reply:
I thought we said it would be okay to remove? Though in retrospect I do think it's probably not good to allow this.